products/chef-infra-client.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Chef Infra Client
+category: server-app
+tags: ruby-runtime
+iconSlug: chef
+permalink: /chef-infra-client
+alternate_urls:
+-   /chef-client
+versionCommand: chef-client -v
+releasePolicyLink: https://docs.chef.io/versions/
+changelogTemplate: "https://docs.chef.io/release_notes_client/__LATEST__"
+activeSupportColumn: true
+releaseDateColumn: true
+
+auto:
+  methods:
+  - custom: chef-infra-client
+
+identifiers:
+-   repology: chef-client
+# eol(x) = releaseDate(x+2) > confirm EOL date here https://docs.chef.io/versions/
+# support(x) = releaseDate(x+1)
+releases:
+-   releaseCycle: "18"
+    releaseDate: 2022-01-10
```

Comment:
18.0 shipped in Oct 2022: https://discourse.chef.io/t/chef-infra-client-18-0-169-released/21570

Reply:
updated